content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -143,7 +143,7 @@ The schema does not explicitly define referential actions on the mandatory `auth
 
 The following caveats apply:
 
-- Referential actions are **not** supported on [implicit many-to-many relations](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations). To use referential actions, you must define an explicit many-to-many relation and define your referential actions on the [join table](/concepts/components/prisma-schema/relations/troubleshooting-relations#how-to-use-a-relation-table-with-a-many-to-many-relationship).
+- Referential actions are **not** supported on [implicit many-to-many relations](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations). To use referential actions, you must define an explicit many-to-many relation and define your referential actions on the [join table](/concepts/components/prisma-schema/relations/troubleshooting-relations#how-to-use-a-relation-table-with-a-many-to-many-relationship). The default referential actions for [implicit many-to-many relations](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations) are `Cascade` both for `onDelete` and `onUpdate` foreign key constraints.
```

Comment:
Puh, this is hard to explain. My try:
```suggestion
- Referential actions are **not** supported on [implicit many-to-many relations](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations). To use referential actions, you must define an explicit many-to-many relation and define your referential actions on the [join table](/concepts/components/prisma-schema/relations/troubleshooting-relations#how-to-use-a-relation-table-with-a-many-to-many-relationship). 
  - The implicit join table in [implicit many-to-many relations](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations) uses `Cascade` for both referential actions `onDelete` and `onUpdate` for both relations. When one of the involved scalar relation fields is updated or deleted, the entry or row in the join table is also updated or deleted.
```

Reply:
I like @janpio's suggestion... some wording tweak suggestions below:

```suggestion
- You can not specify the type of referential action on [implicit many-to-many relations](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations). To specify the referential action, you must instead define an explicit many-to-many relation and define your referential actions on the [join table](/concepts/components/prisma-schema/relations/troubleshooting-relations#how-to-use-a-relation-table-with-a-many-to-many-relationship). 
  - The implicit join table in [implicit many-to-many relations](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations) uses `Cascade` for both `onDelete` and `onUpdate` referential actions. When one of the involved scalar relation fields is updated or deleted, the entry or row in the join table is also updated or deleted.
```

Reply:
Like your iteration @keerlu